cgmanifest.json

```diff
@@ -8176,6 +8176,16 @@
         }
       }
     },
+    {
```

Comment:
This entry should not be needed because the `kernel-uki` package doesn't actually use the kernel source tarball.

Reply:
Ah this makes sense. Thank you!